airgun/widgets.py

```diff
@@ -212,3 +212,53 @@ def fill(self, value):
         self.open_filter.click()
         self.filter_criteria.fill(value)
         self.filter_content.fill(value)
+
+
+class CustomParameter(Widget):
+    """Name-Value paired input elements which can be added, edited or removed.
+
+    Example html representation::
+
+        <table class="table" id="global_parameters_table">
+            <tr class="fields " id="new_os_parameter_row">
+                <input placeholder="Name" type="text" ... id="..._name">
+                <textarea id="new_os_parameter_value" placeholder="Value" ...>
+
+    Locator example::
+
+        //input[@placeholder='Name']
+        //textarea[@placeholder='Value']
+
+    """
+    add_new_value = Text("//a[contains(text(),'+ Add Parameter')]")
```

Comment:
Can we use relative paths, aren't all the objects located at least inside the same view?